openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -205,7 +205,7 @@ def readODB(*argv):
 		return nodes, elements
 
 
-def saveFiberData2D(ModelName, LoadCaseName, eleNumber, sectionNumber, deltaT = 0.0):
+def saveFiberData2D(ModelName, LoadCaseName, eleNumber, sectionNumber, deltaT = 0.0, ZLE = True):
```

Comment:
ZLE=True does not use "sectionNumber" as input (line 234). What section number should the user provide in that case? Does it make sense to make that input optional with a default = 1?

Reply:
In this case there should be no section, as for zero length elements (ZLE) there is no section! For some reason, the reorder is different for zero length elements, I made a test to confirm this one.

Reply:
I get that the recorder is different and there is no section for ZLE. But it looks like there has to be a user input for ``sectionNumber`` since it is not an optional argument. As a user, should I just put a random number there? for example,

`saveFiberData2D("Building","Pushover", 1, 1000)`

The above command will record fiber data of element 1 assuming a ZLE (default), irrespective of the sectionNumber I assign. Also, I think the default value for ZLE should be "False" so the users switch it to "True" only if they record a ZLE. 

`def saveFiberData2D(ModelName, LoadCaseName, eleNumber, sectionNumber=1, deltaT=0.0, ZLE="False")`

The uses for the above definition will be,

`saveFiberData2D("Building","Pushover", 1, ZLE="True)`   To record a ZLE element and,

``saveFiberData2D("Building","Pushover", 1)`   To record a default section (1) in a NL beam-column element.

``saveFiberData2D("Building","Pushover", 1, 4)`   To record a section at integration point 4 in a NL beam-column element.

Reply:
Yeah that is a good point, I think that makes sense on both accounts. ZLE should have equaled False.
I have made the proposed change.